src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -212,6 +212,12 @@ public void verifyInstructorDetails(int instrNum, String newName, String newEmai
         assertEquals(newRole, getInstructorAccessLevel(instrNum));
     }
 
+    public void verifyInstructorDetailsAndStatus(int instrNum, String newName, String newEmail,
+            boolean newIsDisplayedToStudents, String newDisplayName, String newRole, String expectedStatus) {
+        verifyInstructorDetails(instrNum, newName, newEmail, newIsDisplayedToStudents, newDisplayName, newRole);
+        verifyStatus(expectedStatus);
+    }
```

Comment:
* this method can be in the `InstructorCourseEditPageUiTest.java`

* `verifyInstructorDetailsAndStatus(...)` -> `verifyInstructorEdited(...)` (no need for the name to contain all details of the method).

* `instrNum` -> `instructorIndex` (prefer to use full words)

* No need to take the status as a parameter. Use this method only for success cases (i.e., the status message is always the same). That way, the call site can be less verbose.
```java
verifyInstructorEdited(1, "CS2105 Instructor 1", "new_email@email.tmt",
                false, "CS2105 Instructor 1", "Co-owner");
```
In fact you can consider this:
```java
private void verifyInstructorEditSuccess(...){
    courseEditPage.editInstructor(...);
    verifyInstructorDetails(...);
    verifyStatus(...);
}
```